stack.yaml

```diff
@@ -20,9 +18,16 @@ extra-deps:
   - git:
       https://gitlab.com/morley-framework/morley.git # CI can't use SSH
     commit:
-      3bc23ad17a0719ee96d83a94b0194ca5bfe3b8c7 # morley-1.6.0
+      2d0506578493fec3851d711a5cc26c9dc5885001 # past morley-1.7.0
```

Comment:
We put either branch or tag name here. AFAIU this commit is literally morley-1.7.0 tag, so you can remove "past". If you change it to a newer commit that doesn't have a tag, you can write "master" (as long as it's in master).